README.md

```diff
@@ -1,81 +1,87 @@
-# How to register dApp to Cardano + submit CIP-72 format metadata
+# How to register dApp to Cardano
 
-## Set up cardano-cli
+Dapp registration is a transaction with specific metadata describing your dapp.
+For more details refer to [cip-72 specification](https://developers.cardano.org/docs/governance/cardano-improvement-proposals/cip-0072/).
 
-[https://github.com/piotr-iohk/cardano-up](https://github.com/piotr-iohk/cardano-up)
+This repository consists of scripts that intend to help you with submitting the registration transaction.
 
-## Set up local node or use blockfrost.io
+## Set up required tooling
 
-1. For local node setup -> [https://github.com/piotr-iohk/cardano-up](https://github.com/piotr-iohk/cardano-up)
-2. For getting a blockfrost key -> [https://blockfrost.io/](https://blockfrost.io/)
+You will need a node.js and yarn installed in your system.
 
-## Set up wallet
+You need a few tools to be available in your terminal:
+ - `cardani-cli`
+ - `cardano-address`
+ - `bech32`
```

Comment:
A few lines below there are two recommended ways to get the required tools (cardano-wallet or Daedalus). Probably you could get them in plenty of other ways but I don't think we should list more ways. Many of developers are haskelers so it will be obvious for them to use `cbal`

IMO we also should not add a note for them to test whether those tools work. Isn't that obvious to run the tool to test it? 😉 In the section recommending cardano-wallet or Daedalus it is described to set the $PATH env variable properly, so either all of them will work on none.